### PR TITLE
Updates from 3.14

### PR DESCRIPTION
**Pull request by mzohreva**

### CODE REVIEW

**Comment by Goirad** [comment]
I assume this is just the diff from the new auto-generated code?

**Comment by mzohreva** [comment]
> I assume this is just the diff from the new auto-generated code?

Yes

File: src/generated/apps_generated.rs [hunk 1]
```diff
@@ -49,6 +50,7 @@ pub enum AppCredential {
     Secret(String),
     Certificate(Blob),
     TrustedCa(TrustAnchor),
+    GoogleServiceAccount {},
```
Comment (Goirad) [inline src/generated/apps_generated.rs:53]: Why is this an empty struct?

Comment (mzohreva) [reply src/generated/apps_generated.rs:53]: To be able to expand it in the future. You can also check the source that this code was generated from.

**Comment by mzohreva** [comment]
bors retry

**Comment by mzohreva** [comment]
bors retry

**Pull request merged**